client/src/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Menu extends React.Component {
+	constructor() {
```

Comment:
Class components should always call the base constructor with props.  super() should be modified too.
```suggestion
	constructor(props) {
```